src/SFA.DAS.AssessorService.Application.Api/Controllers/StandardServiceController.cs

```diff
@@ -69,5 +69,16 @@ public async Task<IActionResult> GetStandardOptionsForStandard(string id)
             var standardOptions = await _standardService.GetStandardOptionsByStandardId(id);
             return Ok(standardOptions);
         }
+
+        [HttpGet("standard-options/{standardReference}/{version}", Name = "GetStandardOptionsByStandardReferenceAndVersion")]
+        [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(StandardOptions))]
+        [SwaggerResponse((int)HttpStatusCode.BadRequest, Type = typeof(IDictionary<string, string>))]
+        [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
+        public async Task<IActionResult> GetStandardOptionsForStandardReferenceAndVersion(string standardReference, decimal version)
```

Comment:
Same as mentioned previously, the draft specification isn't explicit in mentioning decimal or string